template-LEAS/prompts.md

```
@@ -0,0 +1,64 @@
+USANDO ChatGPT 5
+
+Necesito preparar un prompt para chatgpt para que genera una pagina web sencilla que tendran un stop-watch como la imagen te de adjunto. La pagina no debe tener ninguna dependecia a librerias javascript y no debe contener estilos css inline. La pagina se debe estructurar de la siguiente forma: 1. index.html donde estara el codigo html 2. index.js donde estara el codigo JS que manejara el stop-watch 3. styles.css que tendra los estilos necesarios para que se vea la pagina como la imagen que te adjunto
+
+#PROMPT GENERADO POR ChatGPT
```

markdownlint-cli2 (0.18.1)

No space after hash on atx style heading

(MD018, no-missing-space-atx)